src/test/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtilTest.java

```diff
@@ -379,6 +391,36 @@ public void testEscapeAllControlChars() {
                 JavadocUtil.escapeAllControlChars("1\\r2\\n3\\t"), "invalid result");
     }
 
+    @Test
+    public void testGetContentOfInlineCustomTag() throws CheckstyleException, IOException {
+        final File testFile = new File("src/test/resources/com/"
+                + "puppycrawl/tools/checkstyle/utils/javadocutil/"
+                + "InputJavadocUtilTest_ContentOfInlineCustomTag.java");
+        final DetailAST root = JavaParser.parseFile(testFile, JavaParser.Options.WITH_COMMENTS);
+        final DetailAST javadocCommentAST = root.getLastChild().getFirstChild().getFirstChild()
+                .getNextSibling();
+        final JavadocDetailNodeParser detailNodeParser = new JavadocDetailNodeParser();
+        final DetailNode node = detailNodeParser
+                .parseJavadocAsDetailNode(javadocCommentAST).getTree();
+        final String actual = JavadocUtil.getContentOfInlineCustomTag(node.getChildren()[5]);
+        final String expected = " This is the summary of the class.\n"
```

Comment:
I added whitespace before "This" but I feel it's not needed, in that case just comment not needed and I will remove it.